rust/src/rfb/logger.rs

```diff
@@ -67,12 +67,13 @@ fn log_rfb(tx: &RFBTransaction, js: &mut JsonBuilder) -> Result<(), JsonError> {
         _ => (),
     }
     if let Some(security_result) = &tx.tc_security_result {
+        let result_key = "security_result";
         let _ = match security_result.status {
-            0 => js.set_string("security_result", "OK")?,
-            1 => js.set_string("security-result", "FAIL")?,
-            2 => js.set_string("security_result", "TOOMANY")?,
+            0 => js.set_string(result_key, "OK")?,
+            1 => js.set_string(result_key, "FAIL")?,
+            2 => js.set_string(result_key, "TOOMANY")?,
```

Comment:
This is a breaking change in a patch release, I've been away, but has there been some prior discussion on this?

@catenacyber I don't see any notes in the upgrade guide for 7 to 8 as well, was this change made there as well?

Reply:
For me this is a fix to always use the same key

Reply:
Confirmed.. While technically it is changing behaviour, it is correcting a bug.

Reply:
Correcting a bug is not the question. It's a change of output therefore, it needs documentation on the change.

Reply:
Actually for 7.0.x, there is a clear changelog entry and ticket describing this. I'm more concerned about 8 where there isn't as it was part of another change, so remains undocumented from what I can see. @catenacyber Can you add an upgrade note for 8?

Reply:
Indeed, this was an accidental fix, I propose :
https://github.com/OISF/suricata/pull/11636